cucumber/authenticators_gcp/features/authn_gcp_basic_host.feature

```diff
@@ -0,0 +1,41 @@
+Feature: GCP Authenticator - Hosts can authenticate with GCP authenticator
+
+  In this feature we define an Azure authenticator in policy and perform authentication
+  with Conjur, using a host with subscription-id & resource-group annotations.
+  In successful scenarios we will also define a variable and permit the host to
+  execute it, to verify not only that the host can authenticate with the Azure
+  Authenticator, but that it can retrieve a secret using the Conjur access token.
+
+  Background:
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-gcp
+      body:
+      - !webservice
+
+      - !group apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I have host "test-app"
+    And I set GCP annotations to host "test-app"
```

Comment:
I think you should say exactly which annotation you are setting and it should not be part of the background step